src/dev/run_i18n_check.ts

```diff
@@ -121,11 +123,13 @@ run(
                   ignoreIncompatible: !!ignoreIncompatible,
                   ignoreUnused: !!ignoreUnused,
                   ignoreMissing: !!ignoreMissing,
+                  // By default ignore missing formats
+                  ignoreMissingFormats: ignoreMissingFormats !== false,
                   fix,
                 },
                 log
               ),
-              { exitOnError: true }
+              { exitOnError: false }
```

Comment:
This seems changed the existing behavior, is it by intention?

Reply:
Yeah. If `true`, it exits at the first error. When false, it logs them all and then finally throws.

While "checking" we would rather have all of the problems shown to us, as opposed to just the first problem.